book/types_of_data.md

```diff
@@ -277,41 +277,42 @@ To make a copy of a record with new fields, you can use the [spread operator](/b
 
 ## Lists
 
-Lists are ordered sequences of data values. List syntax is very similar to arrays in JSON. However, commas are _not_ required to separate values if Nushell can easily distinguish them!
+A list is an ordered sequence of values. Unlike most languages, comma separators are optional.
 
 ```nu
-> [sam fred george]
+> [sam, fred, george]
```

Comment:
Commas are optional, yet you show them in the example. I'd rather not show them here.